packages/components/tests/dummy/app/templates/components/link-to/standalone.hbs

```diff
@@ -23,46 +23,7 @@
     <em>Note: Solutions for LinkTo (Inline) are coming soon.</em>
```

Comment:
For some reason, can't add suggestion. How about removing this "Note: Solutions for LinkTo (Inline) are coming soon." and/or adding a link to LinkTo (CTA) page before merge? Non-blocking suggestion.

This is an argument to have Percy do the full page, it makes the PR easier to grok, when there's so many changes